modules/packages/ZMQ.chpl

```diff
@@ -246,6 +246,8 @@ module ZMQ {
   private use IO;
   private use SysCTypes;
   private use CPtr;
+  private use ByteBufferHelpers;
+  private use DSIUtil;
```

Comment:
Let's move these `use`s into `parallelCreateBytesWithNewBuffer` (that way we don't forget to remove them when parallelCreateBytesWithNewBuffer eventually goes away.